client/src/Game.cpp

```diff
@@ -84,31 +68,36 @@ void Game::processEvents() {
 }
 
 void Game::update(sf::Time deltaTime) {
-    player.update(deltaTime);
-
-    playerCommands.updatePlayer(deltaTime);
+    if (!player.expired()) {
```

Comment:
Globally, I think we should start thinking about a state machine to run our application.
 
Something like loading state, then playing state (trggered when we receive the player creation message), etc...